tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedReadHandleImplV2.java

```diff
@@ -123,8 +125,13 @@ public LedgerMetadata getLedgerMetadata() {
 
     @Override
     public CompletableFuture<Void> closeAsync() {
-        CompletableFuture<Void> promise = new CompletableFuture<>();
+        if (closeFuture.get() != null || !closeFuture.compareAndSet(null, new CompletableFuture<>())) {
+            return closeFuture.get();
+        }
+
+        CompletableFuture<Void> promise = closeFuture.get();
         executor.execute(() -> {
+
```

Comment:
Remove this blank and help add at line 153
```
if (log.isDebugEnabled()) 
```

Reply:
@Technoboy- fixed